governance-exercises/DAO Masons poll-exercise-03.md

```diff
@@ -0,0 +1,39 @@
+One of the assumptions presented by the exercise is:
+
+> Currently, one standing order is in force, relating to transfers from the trust to the PSMs and back depending on the amount of collateral in the PSMs.
+
+Emilia begins her message:
+
+> As discussed with large MKR governance holders recently, we are requesting authorization via poll for the following standing order, relating to the transfer of RWA revenues from vaults arranged by MagicRWA.
+
+She includes a link to the standing order and we see it is 'internally consistent with some ambiguity. Covers revenue return only'
+
+I'm noting that the new standing order covers 'revenue return only' which represents a loss of critical function relative to the previous standing order. That needs to be resolved before it is approved.
+
+For that reason I'm holding the new standing order as an incomplete, working draft.
+
+Emilia states:
+
+> The existing standing order related to ALM transfers to maintain the DAI peg is invalidated as a result of this standing order. The ALM order will be replaced shortly once the new version has been fully drafted.
+
+Her use of the present tense seems inappropriate, because the standing order authorized by the DAO isn't invalidated yet. That would require a MakerDAO Proposal.
```

Comment:
That one was likely just me doing a bad job on the exercise. Still, this is a good thing to look out for.